examples/market_maker/on_chain_market_maker.v.py

```diff
@@ -24,10 +24,10 @@ def initiate(token_addr: address, token_quantity: currency_value):
 @public
 @payable
 def eth_to_tokens():
-    fee: wei_value = msg.value / 500
+    fee: wei_value = floor(msg.value / 500)
```

Comment:
@DavidKnott I think we chose the write option in specifying floor or ceil - much clearer which operation we are using now.